### ISSUE TITLE
Why n / ws, h_dst in WeightedSAGEConv in your pinsage tutorial?

### ISSUE THREAD

**Issue by hojun0705** [dmlc/dgl#3725]
In dgl-master/examples/pytorch/pinsage/layers.py,

z = self.act(self.W(self.dropout(torch.cat([n / ws, h_dst], 1))))  from WeightedSAGEConv in your pinsage example.

Here especially, why n / ws? If I understand correctly, n is neighborhood (+ self loop) embedding and ws (sum of the weights). But why n / ws? if weight is higher, the value of n / ws will be smaller. shouldnt it be the other way around like when we have more important weight (higher value), n / ws should be higher? Can you please explain why you did what you did? Thank you.

**Comment by BarclayII** [comment]
Because the messages and weights are aggregated with `sum`, essentially `n / ws` serves as a weighted average of messages, i.e. the value will bias towards messages with larger weights.
